bundles/io/org.eclipse.smarthome.io.rest.sitemap/src/main/java/org/eclipse/smarthome/io/rest/sitemap/internal/PageChangeListener.java

```diff
@@ -213,8 +213,13 @@ private boolean definesVisibility(Widget w, String name) {
         return false;
     }
 
-    public void sitemapContentChanged(EList<Widget> widgets) {
-        updateItemsAndWidgets(widgets);
+    public void sitemapContentChanged() {
+        for (SitemapSubscriptionCallback callback : distinctCallbacks) {
+            SitemapChangedEvent changeEvent = new SitemapChangedEvent();
```

Comment:
Does it really make sense to create a new event for each callback and not create it once and send it to all of them?